src/api/app/components/sourcediff_component.html.haml

```diff
@@ -4,7 +4,7 @@
       .clearfix.mb-2.text-center
         .btn.btn-outline-primary.cache-refresh{ title: 'Refresh results', onclick: "loadChanges(#{@bs_request.number}, #{@action[:id]});" }
           Crunching the latest data. Refresh again in a few seconds
-          %i.fas.fa-sync-alt{ id: "cache#{@index}-reload" }
+          %i.fas.fa-sync-alt{ id: "cache#0-reload" }
```

Comment:
```suggestion
          %i.fas.fa-sync-alt{ id: "cache-reload" }
```
I find the `#0` confusing. Do we really need it?

Reply:
Yes, it's the handler of the first item to be reloaded when the data crunching ends, so 0. If we had many more items to reload they would be `cache#0-reload`, `cache#1-reload`, `cached#2-reload`, etc... So, yes, the zero is needed.

Reply:
See https://github.com/openSUSE/open-build-service/blob/ed980b18d481c36b5e1b420449284c884dea9bae/src/api/app/assets/javascripts/webui/buildresult.js#L6

Reply:
I see we need the index for `'#rpm'+index+'-reload'` elements. But we are now removing the `index` variable from `'#cache'+index+'-reload'` elements.

Is there some other place in the codebase where `'#cache'+index+'-reload'` elements with the `index` variable different than `0` are created or used?

Reply:
[No](https://github.com/search?q=repo%3AopenSUSE%2Fopen-build-service+cache+-reload&type=code)

Reply:
> [No](https://github.com/search?q=repo%3AopenSUSE%2Fopen-build-service+cache+-reload&type=code)

Interesting.

In this PR we are removing the `index` variable from one component: `app/components/sourcediff_component.html.haml`. And we are leaving it on the other one: `app/components/sourcediff_tab_component.html.haml`.

Isn't it safe to remove it from the `sourcediff_tab_component` component too? Is this planned in a follow up pull request?

If there isn't some other place in the codebase where '#cache'+index+'-reload' elements with the `index` variable different than `0` are created or used, I would prefer to use `cache-reload` instead of `cache#0-reload`.

Reply:
> Isn't it safe to remove it from the `sourcediff_tab_component` component too? Is this planned in a follow up pull request?

Probably, right now I'm focused on this component. I will deal with `SourcediffTabComponent` elsewhere.

Reply:
I hope that later on, when you deal with `SourcediffTabComponent` the `cache#0-reload` string could be renamed to something more meaningful.